api/api/clubs/tests.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+
+
+class ListClubsViewTest(APITestCase):
+    """Tests ListClubView for getting the list of clubs."""
+
+    def testListClubs_noClubExistsInDb_returnsEmptyList(self):
+        url = '/clubs/'
```

Comment:
1. `url` can be inlined.
2. add `given, when, then`.

For reference look at this file:
https://github.com/BitByte-TPC/gymkhana/blob/a1c20855145975831bfc892ffcafe33f2d137ae4/api/api/auth/tests.py#L27